imports/ui/pages/components/Button/ExportPgnButton.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { compose } from "redux";
+import injectSheet from "react-jss";
+import { translate } from "../../../HOCs/translate";
+
+const styles = {
+  pgnIcon: {
+    width: "25px",
+    height: "25px"
+  }
+};
+
+const ExportPgnButton = ({ id, translate, classes }) => (
+  <a href={"export/pgn/history/" + id} className="pgnbtn">
```

Comment:
It still uses global styles for `.gamehistory .pgnbtn` and `.gamehistory .pgnbtn:hover` selectors.
Please move those styles into this component.
Replace all occurrences of `<a href={"export/pgn/history/" + game.id} className="pgnbtn">` with `ExportPgnButton`.
Remove global styles for `.gamehistory .pgnbtn` and `.gamehistory .pgnbtn:hover` selectors in `imports/ui/pages/RightSidebarTabs/style.css/